java/src/org/openqa/selenium/remote/http/ClientConfig.java

```diff
@@ -40,7 +39,7 @@ public class ClientConfig {
   private final Proxy proxy;
   private final Credentials credentials;
   private final SSLContext sslContext;
-  private HttpClient.Version version;
+  private String version;
```

Comment:
It would be nice to have a more descriptive name for the variable name since we do not have any JavaDocs that could explain more. 

Something like `httpVersion` should do.

Reply:
Could we make it of java.net.http.HttpClient.Version type? String might be confusing as it is not clear which value to provide unless it is going to fail in runtime at valueOf call

Reply:
If we use a type the config would be coupled to a specific implementation, right?

Reply:
could subclass the config, create a public setter/getter that takes a HttpClient.Version but stores it as a string.

Reply:
Yes, you already have `AppiumClientConfig`, that can have a setter/getter that takes HTTPClient.Version